### PR TITLE
Detailed anime info page

### PR DESCRIPTION
**Pull request by MemboLembo**
- Changed tab space
- Added and stylized detailed anime info page #10

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,81 +1,163 @@
...
+      return title;
+    }
+    return ''
+  };
+
+  function formatDate(dateWithWrongFormat) {
+    if (dateWithWrongFormat) {
+      const year = new Date(dateWithWrongFormat).getFullYear();
+      const mounth = new Date(dateWithWrongFormat).getMonth();
+      const day = new Date(dateWithWrongFormat).getDay();
```
Comment (Badrpas) [inline js/index.js]: Creating date instances via `new Date(dateWithWrongFormat)` each time is redundant.

File: js/index.js [hunk 2]
```diff
@@ -1,81 +1,163 @@
...
+      //   const isLast = objects.length === i + 1 
+      //   return nameList + object.name + (isLast? '' : ', ');
+      // }, '');
+    }
+    return '-'
+  };
+
+  function checkVideoExistence(videoUrl) {
+    if (videoUrl) {
+      return `<iframe width="560" height="315" src="${videoUrl.replace('autoplay=1','autoplay=0')}" frameborder="0" allow="accelerometer; autoplay = 0; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>Your browser does not support iframes!</iframe>`
```
Comment (Badrpas) [inline js/index.js]: Why single line.

**Review by Badrpas — CHANGES_REQUESTED** [review]

File: js/index.js [hunk 3]
```diff
@@ -1,81 +1,163 @@
...
+    return '-'
+  };
+
+  function getNameListFromObjects(objects) {
+    if (objects && objects.length !== 0) {
+      return objects.map(o => o.name).join(', ');
+      // return objects.reduce((nameList, object, i) => {
+      //   const isLast = objects.length === i + 1 
+      //   return nameList + object.name + (isLast? '' : ', ');
+      // }, '');
```
Comment (Badrpas) [inline js/index.js]: Please remove commented lines

**Review by Badrpas — CHANGES_REQUESTED** [review]

File: js/index.js (hunk 3 above)
Comment (MemboLembo) [reply js/index.js]: All your remarks have been corrected in commit that are below

**Pull request merged**